pkg/zonemgr/zone_manager.go

```diff
@@ -28,52 +28,26 @@ func NewZoneManager() *ZoneManager {
 }
 
 func (zoneMgr *ZoneManager) init() {
+	domain := os.Getenv("DOMAIN")
+	nameServerIP := os.Getenv("NAME_SERVER_IP")
 
-	fmt.Println("================ inside ZoneManager.init ===============")
+	zoneMgr.zoneFileCache = NewZoneFile(nameServerIP, domain)
+	zoneMgr.zoneFileCache.init()
 
-	//todo fetch custom details
-	clusterName := "cluster"
-	zoneFileName := "/zones/db.secdns"
-
-	userAdminEmail := ""
-	userSubdomain := "domain.com"
-	userNameserverName := ""
-	userNameserverIP := ""
-
-	zoneMgr.zoneFileCopy = NewZoneFile(clusterName, userAdminEmail,
-		userNameserverName, userNameserverIP, userSubdomain)
-
-	zoneMgr.zoneFileCopy.init()
-
-	//fmt.Println("================ zoneMgr.zoneFileCopy.content ===============\n", zoneMgr.zoneFileCopy.content)
-
-	zoneMgr.zoneFileDns = NewDnsZoneFile(zoneFileName)
-	// todo return zoneFileDns.writeFile(zoneFileCopy.content) // is it necessary to create an empty zone file on load??
+	zoneMgr.zoneFileDns = NewDnsZoneFile(zoneFileName + zoneMgr.zoneFileCache.domain)
```

Comment:
file name imo should be constant, unless you find it is a must that it is according ORIGIN
and then we can later investigate how to change the Corefile so it will be agnostic

Reply:
From what I saw, file name has to match ORIGIN value otherwise dns wouldn't load it properly. 
So I implemented it this way for now to make it work, I'll check later if the dependency can be avoided somehow

Reply:
Indeed the `{1}` tells it to look for extension origin
https://coredns.io/plugins/auto/

well we can keep it this way